server/controllers/routes/user/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,21 @@
+const {
+  cohort: { getSpecificCohort },
+} = require('../../../../database/queries');
+
+exports.getSpecificCohort = async (req, res, next) => {
+  try {
+    const { cohortid } = req.params;
+    const { rows } = await getSpecificCohort(cohortid);
+    const data = { ...rows[0] };
+    if (data.id) {
+      res.status(200).json({ statusCode: 200, data });
+    } else {
+      res.status(400).json({
```

Comment:
@Mu7ammadAbed If you want, you can be more specific with the error code and return `404` (not found) instead of `400` here.  That would be good because then you could use `400` for other, more serious error cases (and then the frontend can understand more about the error that happened).